fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/election/fragments/ElectionFragment.java

```diff
@@ -49,11 +58,19 @@ public class ElectionFragment extends Fragment {
   private final EnumMap<EventState, Integer> actionTextMap = buildActionTextMap();
   private final EnumMap<EventState, Boolean> actionEnablingMap = buildActionEnablingMap();
 
-  public static ElectionFragment newInstance() {
-    return new ElectionFragment();
-  }
+  private final CompositeDisposable disposables = new CompositeDisposable();
 
-  public ElectionFragment() { // required empty constructor
+  @Inject ElectionRepository electionRepository;
```

Comment:
Well that's better than what I did (creating an intermediary method in the viewModel)